recettes/sucrees/tarte_citron_meringuee.md

```diff
@@ -0,0 +1,47 @@
+# Tarte au citron meringuée
+
+---
+
+### Ingrédients
+
+- [ ] 1 (pâte sablée)[./pate_sablee.md]
+
+###### Crème au citron
+
+- [ ] 180g sucre
+- [ ] 2 citrons
+- [ ] 4 oeufs
+- [ ] 20cl crème liquide
+
+###### Meringue
+
+- [ ] 2 oeufs
+- [ ] 150g sucre glace
+
+---
+
+### Recette
+
+- Préparer la pâte sablée en utilisant 2 jaunes plutôt qu'un oeuf entier. Réserver les 2 blancs restant pour la meringue
+- Faire pré-cuire la pâte 10min à 180°C
+
+###### Crème au citron
+
+- Mélanger 2 oeufs entiers et 2 jaunes (réserver les 2 blancs restant)
+- Ajouter la crème fraiche et le sucre
+- Ajouter le zeste des citrons ainsi que leur jus
+- Bien battre pour homogénéiser le tout et verser le mélange sur la pâte pré-cuite
+- Cuisson 30min à 180°C
+
+###### Meringue
+
+- Battre les 4 blancs d'oeufs restants neige
+- Ajouter le sucre glace petit-à-petit en continuant de battre
```

Comment:
pas sûre qu'il y a des tirets dans petit à petit, can you verify?

Reply:
Yup that's right bro